scripts/BuildAndTest.ps1

```diff
@@ -30,8 +30,8 @@
     Do not run dotnet publish, which creates a layout directory.
 .PARAMETER NoSigningDirectory
     Do not create a directory containing the binaries that need to be signed.
-.PARAMETER Install
-    Install the VSIX.
+.PARAMETER Associate
```

Comment:
I was thinking this could still be a scenario. Without the vsix, it allows sarif files to be opened in VS for json editing

---
In reply to: [216406108](https://github.com/Microsoft/sarif-sdk/pull/1078#discussion_r216406108) [](ancestors = 216406108)

Reply:
And the Install option is already present in the viewer repo.

---
In reply to: [216406925](https://github.com/Microsoft/sarif-sdk/pull/1078#discussion_r216406925) [](ancestors = 216406925,216406108)

Reply:
Ok. Please add comment explaining this.

---
In reply to: [216409873](https://github.com/Microsoft/sarif-sdk/pull/1078#discussion_r216409873) [](ancestors = 216409873,216406925,216406108)